lib/shopify_app/controller_concerns/login_protection.rb

```diff
@@ -223,7 +223,6 @@ def sanitized_params
     end
 
     def return_address
-      return base_return_address unless ShopifyApp.configuration.allow_jwt_authentication
```

Comment:
What's the rationale behind this change?

Reply:
the return address previously didn't have the `shop` and `host` params when redirecting to the app after OAuth. Because of that, we're not able to use those params to load App Bridge to embed the app inside of an iframe. We can simply pass in the params in the return address now.

Reply:
Ok, can you help me understand how this is related to the omniauth upgrade?

Reply:
This isn't actually. I've gone ahead and created a [new PR](https://github.com/Shopify/shopify_app/pull/1259) for this. Could I get a review on that?